gradle/maven.gradle

```diff
@@ -1,5 +1,5 @@
 allprojects {
-  version = "4.0.0"
+  version = "4.2.0-SNAPSHOT"
```

Comment:
why a snapshot version?

Reply:
Per wiki https://github.com/linkedin/brooklin/wiki/Developer-Guide#update-mavengradle and per what's typical. Master/trunk should be a SNAPSHOT of the next release. Can ignore the intermediate 4.1.0 commit -- I'll squash it anyway.

Full disclosure, I tweaked the wiki there too, cuz it used to say to point master to the next PATCH release (which was wrong).

Reply:
So, while releasing, you will change this version and then make the release so that version 4.2.0 is released. Or the new version will be 4.2.0-SNAPSHOT, which will be referenced everywhere else.

Reply:
Right. During release process you just drop the SNAPSHOT part on your local machine. Then build and publish. Then increment the version to <next.version.0>-SNAPSHOT and submit a PR for the change. I had to do it a bit out of order this time bc the version in master was actually the previous release (should be the next release snapshot).